easybuild/easyconfigs/p/pygame/pygame-2.1.0-GCCcore-11.3.0.eb

```
@@ -0,0 +1,47 @@
+easyblock = 'PythonBundle'
+
+name = 'pygame'
+version = '2.1.0'
+
+homepage = 'https://www.pygame.org'
+description = """
+Pygame is a set of Python modules designed for writing video games. Pygame adds
+functionality on top of the excellent SDL library. This allows you to create
+fully featured games and multimedia programs in the python language.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SDL2_image', '2.6.3'),
+    ('SDL2_mixer', '2.6.3'),
+    ('SDL2_ttf', '2.20.2'),
+    ('PortMidi', '2.0.4'),
+    ('libpng', '1.6.37'),
+]
+
+use_pip = True
+
+_pygame_extra_base = ':'.join([
+    '$EBROOTSDL2_IMAGE',
+    '$EBROOTSDL2_MIXER',
+    '$EBROOTSDL2_TTF',
+    '$EBROOTLIBJPEGMINTURBO',
+    '$EBROOTPORTMIDI',
```

Adding the dependency on `libpng` didn't solve the error for me, but apparently it also has to be added here:

```suggestion
    '$EBROOTLIBPNG',
    '$EBROOTPORTMIDI',
```